index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
![image](https://user-images.githubusercontent.com/74744787/165403972-e18ad399-9f9f-49c0-9d71-b9c66c41bd4a.png)

1- Please provide a meaningful title for your pull request, add linters isn't the main thing you did in this pull request, I think it would be better if you give it a more meaningful name like Hello Microverse.

2- Please Provide a description of this pull request in this description you can mention everything you did in this repo including adding linter.